egs/thchs30/s5/cmd.sh

```diff
@@ -0,0 +1,35 @@
+# "queue.pl" uses qsub.  The options to it are
+# options to qsub.  If you have GridEngine installed,
+# change this to a queue you have access to.
+# Otherwise, use "run.pl", which will run jobs locally
+# (make sure your --num-jobs options are no more than
+# the number of cpus on your machine.
+
+#a) JHU cluster options
+#export train_cmd="queue.pl -l arch=*64"
+#export decode_cmd="queue.pl -l arch=*64,mem_free=2G,ram_free=2G"
+#export mkgraph_cmd="queue.pl -l arch=*64,ram_free=4G,mem_free=4G"
+
+#export cuda_cmd="..."
+
+
+#b) BUT cluster options
+#export train_cmd="queue.pl -q all.q@@blade -l ram_free=1200M,mem_free=1200M"
+#export decode_cmd="queue.pl -q all.q@@blade -l ram_free=1700M,mem_free=1700M"
+#export decodebig_cmd="queue.pl -q all.q@@blade -l ram_free=4G,mem_free=4G"
+
+#export cuda_cmd="queue.pl -q long.q@@pco203 -l gpu=1"
+#export cuda_cmd="queue.pl -q long.q@pcspeech-gpu"
+#export mkgraph_cmd="queue.pl -q all.q@@servers -l ram_free=4G,mem_free=4G"
+
+#c) CSLT cluster options
+
+export train_cmd="queue.pl -q cpu.q"
+export decode_cmd="queue.pl -q cpu.q"
+export cuda_cmd="queue.pl -q cuda.q@grid-11"
+
```

Comment:
It would be better to use the 'new-style' options here.  train_cmd and decode_cmd should just be queue.pl, and cuda_cmd would be queue.pl --gpu 1, and locally you could create a conf/queue.conf to explain how to translate these into options to GridEngine.
See http://kaldi-asr.org/doc/queue.html and search for conf/queue.conf